packages/types/README.md

```diff
@@ -0,0 +1,3 @@
+# Types
+
+Utility package with core TypeScript types used across many other packages.
```

Comment:
Should this state that it's for Web Stories packages?

Reply:
I think that's implicit, other packages also imply this.